bin/common.sh

```diff
@@ -187,9 +187,42 @@ function randomPassword {
   date +%s | sha256sum | base64 | head -c 32 ; echo
 }
 
+function disable_sa_token_automount {
+  local ns sa_name should_disable
+  ns=$1
+  sa_name=$2
+  should_disable=${SEC_DISABLE_SA_TOKEN_AUTOMOUNT:-true}
+
+  if [ "$should_disable" == "true" ]; then
+     log_debug "Disabling automount of API tokens for serviceAccount [$ns/$sa_name]"
+     kubectl -n $ns patch serviceAccount $sa_name -p '{"automountServiceAccountToken":false}'
+  fi
```

Comment:
Add a DEBUG message indicating that serviceAccount will NOT be patched when SEC_DISABLE_SA_TOKEN_AUTOMOUNT is set to "false"?

Reply:
Done